src/map/entities/mobentity.cpp

```diff
@@ -784,7 +784,7 @@ void CMobEntity::OnMobSkillFinished(CMobSkillState& state, action_t& action)
     }
 }
 
-void CMobEntity::DropItems()
+void CMobEntity::HandleDeath()
```

Comment:
can we change the name of this function? It gets queued from `m_dropItemTime` milliseconds after dying, so it's not strictly "handling death" - mostly the distribution of stuff (minus kill all status effects, but that could probably get moved back into `Die`)
something like DistributeRewards, or whatever is probably good